Find a functionally equivalent code:com.android.internal.widget.LockPatternUtils.checkPasswordHistory:COMMENT
Method Modifier: public      internal    
Comment:/**
 * Check to see if a password matches any of the passwords stored in the
 * password history.
 *
 * @param password The password to check.
 * @return Whether the password matches any in the history.
 */

@@ -1,5 +1,5 @@
 {
-    String passwordHashString = new String(passwordToHash(password));
+    String passwordHashString = new String(passwordToHash(password, getCurrentOrCallingUserId()));
     String passwordHistory = getString(PASSWORD_HISTORY_KEY);
     if (passwordHistory == null) {
         return false;

